yarn.lock

```diff
@@ -6894,10 +6894,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"minipass@npm:^4.0.0":
-  version: 4.2.8
-  resolution: "minipass@npm:4.2.8"
-  checksum: 7f4914d5295a9a30807cae5227a37a926e6d910c03f315930fde52332cf0575dfbc20295318f91f0baf0e6bb11a6f668e30cde8027dea7a11b9d159867a3c830
+"minipass@npm:^5.0.0":
```

Comment:
Since this is a major version change, have we checked it's not breaking anything 🤔

Reply:
For non-dev dependencies, the `tar` package is only used by `ssh2` (can check this with `yarn why tar`), which is only used for the Continuous Testing tunnel.

I tested it locally on this PR: https://github.com/DataDog/datadog-ci/pull/1281

The unit tests are passing, and the tunnel is still working:

![image](https://github.com/DataDog/synthetics-ci-github-action/assets/9317502/b7273011-386e-4952-aa50-ed91b468f5d5)